lib/util/DateTimeHelpers.ts

```diff
@@ -0,0 +1,191 @@
+import type {
+  IDateRepresentation, IDateTimeRepresentation, IDayTimeDurationRepresentation,
+  IDurationRepresentation,
+  ITimeRepresentation,
+  ITimeZoneRepresentation, IYearMonthDurationRepresentation,
+} from './InternalRepresentations';
+import { extractDayTimeDur } from './InternalRepresentations';
+import { maximumDayInMonthFor } from './specAlgos';
+
+function numSer(num: number, min = 2): string {
```

Comment:
Not sure what `numSer` means. Could we use a more descriptive name?

Reply:
Number serializer :)